sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -240,7 +238,7 @@ def match(actual):
       assert_that(result, matcher())
       pipeline.run()
```

Comment:
The pipeline creation is in the loop, so the run is as well. However, I'll change it to run a single pipeline with multiple samples instead.